Style changes

hariharan-mahadevan-10884 opened the pull request.

Commit:
Light theme
Changed style for radio buttons
Added style for submit button

Commit:
Added border radius for buttons

Commit:
Made requested changes

Commit:
Changed radio button style

sivaraam committed:
Merge branch 'master' into Color

sivaraam committed:
Fixed white space issues

sivaraam committed:
Added dummy table content (TO BE REMOVED)

sivaraam commented on css/main.css:
```diff
@@ -194,8 +194,8 @@ table {
-td, th {
-  border: 1px solid;
+table th, table tr , table td{
+  border:1px solid;
```
I still don't see any table border. 
![table border](https://cloud.githubusercontent.com/assets/12448084/25321145/ea81ff72-28c9-11e7-82ed-943a25baa1a3.png)

sivaraam commented on css/radio-button-light-style.css:
```diff
@@ -0,0 +1,28 @@
+  border-color: #036;
+  background-color:#036;
+}
+
```
Leave the background for a selected radio button as it was don't color it.

sivaraam commented:
**ALERT :** Before making any changes pull in the changes I made to your local Color branch

sivaraam commented on css/light-styles.css:
```diff
@@ -40,6 +40,25 @@ table {
+  border-color: #c69;
+  background-color:  #c69;
+}
+
```
Don't color the background of table cells

sivaraam commented on css/light-styles.css:
```diff
@@ -40,6 +40,25 @@ table {
+table td{
+  border-color: #c69;
+  background-color:  #c69;
+}
```
Don't color the background of table cells.

sivaraam commented on css/main.css:
```diff
@@ -207,6 +207,25 @@ td, th {
+    -webkit-transition-duration: 0.4s; /* Safari */
+    transition-duration: 0.4s;
+    cursor: pointer;
+}
```
Round the value of ems (in all the places _you_ gave them) to the first decimal or to the nearest integer

sivaraam reviewed (changes requested):
Apart from the inline comments, speccify the changes done in the commit message. Don't specify _Made requested changes_.

Commit:
Style changes
1.Brought table borders back again
2.Removed table background colors
3.Changed Radio button styles
